kratos/includes/define.h

```diff
@@ -349,13 +349,13 @@ catch(...) { Block KRATOS_THROW_ERROR(std::runtime_error, "Unknown error", MoreI
 #undef KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION
 #endif
 #define KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(flag) \
- scope().attr(#flag) = boost::ref(flag)      \
+ scope().attr(#flag) = boost::ref(flag);      \
```

Comment:
Do we need this semicolon? In my opinion we should leave this without and put it when it uses to make format corrector work well.